### PR TITLE
Implement fb login

### PR DESCRIPTION
**Pull request by sebsonfire**
PULL MY SHIT

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -2,5 +2,25 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+  :recoverable, :rememberable, :trackable, :validatable, :omniauthable, omniauth_providers: [:facebook]
```
Comment (jpjagt) [inline app/models/user.rb:5]: I would put devise :omniauthable, omniauth_providers: [:facebook] on a seperate line to improve readability and clearness of what it does.

Comment (sebsonfire) [reply app/models/user.rb:5]: fuck you.

Comment (jpjagt) [reply app/models/user.rb:5]: don't forget who the owner of this github repo is, mate.

Comment (sebsonfire) [reply app/models/user.rb:5]: My bad. What I meant to say was:

WHY DONT YOU LIKE MY BALLS, MR. GARRISON.

Comment (jpjagt) [reply app/models/user.rb:5]: ok ok i'll merge, whatever. jeez

**Pull request merged**